playbooks/troubleshooting/troubleshooting_guide.adoc

```diff
@@ -0,0 +1,719 @@
+
+----
+{
+  "apiVersion": "v1",
+  "kind": "PersistentVolume",
+  "metadata": { "name": "docker-registry-storage"
+  },
+  "spec": { "capacity": { "storage": "50Gi" }, "accessModes": [ "ReadWriteOnce" ], "nfs": { "path": "/storage/vms/ose_nfs/stratus/", "server": "192.168.200.99" }, "persistentVolumeReclaimPolicy": "Recycle"
+  }
+}
+----
+
+----
+[root@master00 ~]# oc get pv
+NAME                      LABELS    CAPACITY   ACCESSMODES   STATUS    CLAIM                          REASON    AGE
+docker-registry-storage   <none>    50Gi       RWO           Bound     default/docker-storage-claim             39m
+----
+
+----
+apiVersion: "v1"
+kind: "PersistentVolumeClaim"
+metadata:
+  name: "docker-storage-claim"
+spec:
+  accessModes:
+    - "ReadWriteOnce"
+  resources:
+    requests:
+      storage: "50Gi"
+----
+
+
+----
+[root@master00 ~]# oc get pvc
+NAME                   LABELS    STATUS    VOLUME                    CAPACITY   ACCESSMODES   AGE
+docker-storage-claim   <none>    Bound     docker-registry-storage   50Gi       RWO           38m
+----
+
+==== Solution
+
+This is most likely an error during the editing of the deployment
+config. When assigning the volume the *--name=* section refers to the
+name of the volume to be overwritten. The template for the router is
+mounting the *registry-storage* volume to **/registry**. Creating new
+claims and volumes thus, will not have the desired effect. You need to
+overwrite the default which is **emptydir** with the PV.
+
+*WRONG COMMAND:*
+
+----
+oc volume deploymentconfigs/docker-registry --add --name=docker-registry-storage -t pvc --claim-name=docker-storage-claim --overwrite
+----
+
+This is overwriting/creating a volume called docker-registry-storage
+which will not be able to be mounted in */registry*
+
+*CORRECT COMMAND:*
+
+----
+oc volume deploymentconfigs/docker-registry --add --name=registry-storage -t pvc --claim-name=docker-storage-claim --overwrite
+----
+
+=== Hosts Can No Longer Resolve Each Other During Anisble Install
+
+During the install, firewalld is disabled in favor of iptables. This
+transition may not open port 53 which is required for DNS lookups.
+
+==== Solution
+
+Use a DNS source external to the installer (at least during the installation
+  phase) so that the installation does not fail due to host name lookup problems.
+  +
+  +
+
+=== Failure to deploy registry (permissions issues)
+
+Deploying a registry requires a user with cluster-admin privileges. The
+system:admin user should have these
+privileges. https://docs.openshift.com/enterprise/3.1/install_config/install/docker_registry.html[OSEv3.1 Deploying a Docker Registry]
+ +
+ +
+
+=== Application Pod fails to deploy
+
+This can be caused by DNS issues. Turn on the debugging on your dns
+server. If using bind:
+
+
+----
+rdnc querylog
+----
+
+
+Then tail */var/log/messages* if you see entries like this:
+
+----
+lb00-jarae.jarae.example.com.jarae.example.com
```

Comment:
I think replacing your username with just "ose.example.com" as the base domain will make more sense.